cub/cub/device/dispatch/tuning/tuning_scan.cuh

```diff
@@ -242,19 +276,19 @@ struct policy_hub
   struct Policy350 : ChainedPolicy<350, Policy350, Policy350>
   {
     // GTX Titan: 29.5B items/s (232.4 GB/s) @ 48M 32-bit T
-    using ScanPolicyT =
+    using ScanPolicy =
```

Comment:
The macro `CUB_DEFINE_SUB_POLICY_GETTER` makes some assumptions about the name of the policy so this rename is necessary.

Reply:
Sooo this is potentially problematic. The macro can be changed, what I'm mildly concerned about here is that this will break users who provide their own policies and spell them `ScanPolicyT`, and that stops working with this PR. cc @gevtushenko 

We should probably just inline the macro above and make this work with the existing name.

Reply:
Agreed, code passing user-defined policy should still work. For instance, this change should brake our scan tuning: https://github.com/NVIDIA/cccl/blob/3155c4bd191ea8dedca5b8dd485446d73ae8ec68/cub/benchmarks/bench/scan/exclusive/base.cuh#L73, we just don't build it as part of CI

Reply:
OK - I reverted this change and inlined the macro as suggested. Thanks!